test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -93,6 +94,9 @@ static void initialize() throws InterruptedException, InvocationTargetException
             public void propertyChange(PropertyChangeEvent evt) {
                 if (JFileChooser.SELECTED_FILE_CHANGED_PROPERTY.equals(evt.getPropertyName())) {
                     System.out.println(String.format("Absolute Path : %s",evt.getNewValue()));
+                    JOptionPane.showMessageDialog(null, evt.getNewValue(),
+                            "Absolute path",
+                            JOptionPane.INFORMATION_MESSAGE);
```

Comment:
I don't think it is a good idea. For each selection, the tester will get a modal dialog box with the path. It could even prevent the tester from navigating directories for testing.

I suggest adding a `JListBox` or `JTextArea` on top or bottom of the `JFileChooser` and add the messages there.

Reply:
Can use JLabel and keep updating(remove the old value and show path which is clicked) it right......? Storing previous paths are not required anyhow.